packages/terra-application-name/src/ApplicationMenuName.module.scss

```diff
@@ -1,4 +1,7 @@
 /* stylelint-disable selector-class-pattern */
+// Themes
+@import './clinical-lowlight-theme/ApplicationName.module';
```

Comment:
We also need to specify the fusion-theme values.

Reply:
Some of the packages have only one of the theme files. I guess I'll have to add the missing file

Reply:
Added 4f672fe